README.md

```diff
@@ -161,20 +188,71 @@ For GCE the absolute path of the service account json file should be provided
       test_var3: "test_var3 msg is grp4 hello"
 
 ```
-## Usage
-### Provision a topology
-```
-command: ansible-playbook -vvv provision/site.yml -e "{'data':'path_to_topology_file', 'schema':'path_to_schema_file', 'state':'present', 'topology_output_file':'path_to_output_file'}"
 
-example: ansible-playbook -vvv provision/site.yml -e "{'data':'../ex_topo/ex_data.yml', 'schema':'../ex_schemas/schema_v2.json', 'state':'present', 'topology_output_file':'/tmp/ex_data_output.yaml'}"
+## Example Inventory Layout (openshift 3 node cluster)
 ```
+---
+inventory_layout:
+  vars:
+    openshift_hostname: __IP__
+    openshift_public_hostname: __IP__
+  hosts:
+    openshift-master:
+      host_groups:
+        - masters
+        - nodes
+        - OSEv3
+    openshift-node:
+      count: 1
+      host_groups:
+        - nodes
+        - OSEv3
+    openshift-repo-host:
+      host_groups:
+        - nodes
+        - OSEv3
+        - repo_host
+  host_groups:
+    OSEv3:
+      vars:
+        openshift_docker_additional_registries: |
+            "registry.example.com"
+        openshift_docker_insecure_registries: |
+            "registry.example.com"
+        debug_level: 2
+        ansible_sudo: False
+        ansible_ssh_user: root
+        openshift_override_hostname_check: True
+        openshift_set_hostname: True
+      children:
+        - masters
+        - nodes
 
-### De-Provision a topology
 ```
-command: ansible-playbook -vvv provision/site.yml -e "{'data':'path_to_topolgy_file', 'schema':'path_to_schema_file', 'state':'absent', 'topology_output_file':'path_to_output_file'}"
+## CLI Examples
+### Provision a topology
 
-example: ansible-playbook -vvv provision/site.yml -e "{'data':'../ex_topo/ex_data.yml', 'schema':'../ex_schemas/schema_v2.json', 'state':'absent', 'topology_output_file':'/tmp/ex_data_output.yaml'}"
 ```
+$ ansible-playbook -vvv provision/site.yml -e "topology='/path/to/topology_file'" \
```

Comment:
@herlo  : Why are there multiple -e args instead of single -e with multiple variables inside it ?